docs/examples/basic/README.md

```diff
@@ -0,0 +1,330 @@
+  - sqlite=3.33.0=h62c20be_0
+  - tk=8.6.10=hbc83047_0
+  - wheel=0.36.2=pyhd3eb1b0_0
+  - xz=5.2.5=h7b6447c_0
+  - zlib=1.2.11=h7b6447c_3
+  - pip:
+    - mlops-tempo
+    - mlserver==0.3.1.dev5
+    - mlserver-tempo==0.3.1.dev5
+```
+
+
+```python
+classifier.save(save_env=True)
+```
+
+### Deploying pipeline
+
+
+```python
+classifier.deploy()
+classifier.wait_ready()
+```
+
+### Sending requests
+
+We can send requests to the deployed components with either the python code for the classifier running locally or remotely. 
+
+First we test calling the classifer locally. It will call out to the remote models running in Docker.
+
+
+```python
+classifier(payload=np.array([[1, 2, 3, 4]]))
+```
+
+Now we can use the `.remote` method to call to the remote classifier running in Docker.
+
+
+```python
+classifier.remote(payload=np.array([[1, 2, 3, 4]]))
+```
+
+
+```python
+classifier.remote(payload=np.array([[5.964,4.006,2.081,1.031]]))
+```
+
+### Undeploy pipeline
+
+
+```python
+classifier.undeploy()
+```
+
+## Deploying pipeline to K8s
+
+The next step, will be to deploy our pipeline to Kubernetes.
+We will divide this process into 3 sub-steps:
+
+1. Save our artifacts and environment
+2. Upload to remote storage
+3. Deploy resources
+
+### Setup Namespace
+
+
+```python
+!kubectl create namespace production
+```
+
+
+```python
+!kubectl apply -f ../../../k8s/tempo-pipeline-rbac.yaml -n production
+```
+
+
+```python
+%%writefile minio-secret.yaml
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: minio-secret
+type: Opaque
+stringData:
+  AWS_ACCESS_KEY_ID: minioadmin
+  AWS_SECRET_ACCESS_KEY: minioadmin
+  AWS_ENDPOINT_URL: http://minio.minio-system.svc.cluster.local:9000
+  USE_SSL: "false"
+```
+
+
+```python
+!kubectl apply -f minio-secret.yaml -n production
+```
+
+### Change runtime
+
+
+```python
+k8s_options = KubernetesOptions(namespace="production",authSecretName="minio-secret")
+
+k8s_runtime = SeldonKubernetesRuntime(k8s_options=k8s_options)
+sklearn_model.set_runtime(k8s_runtime)
+xgboost_model.set_runtime(k8s_runtime)
+
+k8s_runtime_v2 = SeldonKubernetesRuntime(k8s_options=k8s_options, protocol=KFServingV2Protocol())
+classifier.set_runtime(k8s_runtime_v2)
+```
+
+### Saving artifacts
+
+Just save the artfacts and not the environment as well.
+
+
+```python
+classifier.save(save_env=False)
+```
+
+### Uploading artifacts
+
+
+```python
+MINIO_IP=!kubectl get svc minio -n minio-system -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+MINIO_IP=MINIO_IP[0]
+```
+
+
+```python
+%%writetemplate rclone.conf
+[s3]
+type = s3
+provider = minio
+env_auth = false
+access_key_id = minioadmin
+secret_access_key = minioadmin
+endpoint = http://{MINIO_IP}:9000
+
+```
+
+
+```python
+import os
+from tempo.conf import settings
+settings.rclone_cfg = os.getcwd() + "/rclone.conf"
+```
+
+
+```python
+sklearn_model.upload()
+xgboost_model.upload()
+classifier.upload()
+```
+
+### Deploy
+
+
+```python
+classifier.deploy()
```

Comment:
We could add comment that first kubernetes deployment may take a bit of time because it will need to pull docker images.